source/getting-started/quick-start.md

```diff
@@ -74,6 +74,8 @@ Let's create a new template using the `ember generate` command.
 ember generate template application
 ```
 
+Ember CLI will notice that there's already a template there and ask if you want to overwrite it. Enter `y` to continue.
```

Comment:
Removed the instruction to generate the template, and edited the paragraph to reflect the change.